db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Retirement/StateMachines/StackRetirement.class/__methods__/update_retirement_status.rb

```diff
@@ -20,12 +20,20 @@
 
 stack = $evm.root['orchestration_stack']
 
+# Update Status Message
+updated_message  = "Server [#{server.name}] "
+updated_message += "Stack [#{stack.name}] " if stack
+updated_message += "Step [#{step}] "
+updated_message += "Status [#{status}] "
+updated_message += "Current Retry Number [#{$evm.root['ae_state_retries']}]" if $evm.root['ae_result'] == 'retry'
+
 # Update Status for on_error for all states other than the first state which is startretirement
 # in the retirement state machine.
 if $evm.root['ae_result'] == 'error'
   if step.downcase == 'startretirement'
     $evm.log("info", "Cannot continue because stack is already retired or is being retired.")
```

Comment:
@billfitzgerald0120 Can we add a "warn" notification for the case where the VM is already retired or is being retired?